formats/service_manual_service_standard/publisher/details.json

```diff
@@ -5,6 +5,11 @@
   "properties": {
     "body": {
       "$ref": "#/definitions/body"
+    },
+    "poster_url": {
```

Comment:
I'm really not sure that this should be a string – it should probably be an `asset_link` definition, but we'd need to make a bunch of updates in service-manual-publisher for it to work properly. @36degrees – what do you reckon?